karoo_gp/test/test_tree.py

```diff
@@ -3,6 +3,7 @@
 import json
 
 from karoo_gp import Tree, Branch
+from .util import dump_json
```

Comment:
This doesn't seem to be used.

Reply:
It's used on line 92 but normally commented out. Lets you quickly update the tests to check what's wrong or update the baseline.